### PR TITLE
Build release package

### PR DESCRIPTION
**Pull request by wenwei-dev**

### CODE REVIEW

**Comment by AmeBel** [comment]
I think the version should be update to one in https://github.com/opencog/cogutils/blob/master/opencog/util/cogutil.h. 

Maybe cpack macros should be dependent on cogutil.h macros or vice-versa. @linas what do you think?

**Comment by linas** [comment]
Amen, don't over-engineer it.  There might be valid reasons to bump the atomspace version number, and leave the cogutils and opencog versions unchanged.

**Comment by wenwei-dev** [comment]
Hmm, "For simplicity, I'll use the version defined in version.h for atomspace, and cogutil.h for cogutils and leave opencog as it is now, which is 0.1.4. "

File: CMakeLists.txt [hunk 1]
```diff
@@ -303,9 +314,10 @@ SET(CPACK_SOURCE_STGZ "OFF")
 SET(CPACK_SOURCE_TBZ2 "ON")
 SET(CPACK_SOURCE_TGZ "OFF")
 SET(CPACK_SOURCE_TZ "OFF")
+SET(CPACK_PACKAGE_CONTACT "team@opencog.org")
```
Comment (linas) [inline CMakeLists.txt]: I don't think this mailing list exists. I think you want to use <opencog@googlegroups.com> as the contact point.

**Comment by linas** [comment]
This looks reasonable (except for the bad email address) and can be merged. Wenwei, can you update the pull request?

**Pull request merged**